plugin/runtime/status.py

```
@@ -70,10 +70,38 @@ def get_plugin_status(self, plugin_id: Optional[str] = None) -> Dict[str, Any]:
+                "updated_at": _now_iso(),
+                "source": "main_process_synthetic",
+            }
+
+        for pid in registered_plugin_ids:
+            if pid not in cached:
+                cached[pid] = _build_synthetic(pid, "stopped")
+
+        for pid, host in plugin_hosts_snapshot.items():
+            alive = False
+            try:
+                alive = bool(getattr(host, "is_alive") and host.is_alive())
```

Ruff (0.14.10)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)